pyirf/io/gadf.py

```diff
@@ -325,3 +309,177 @@ def create_rad_max_hdu(
     _add_header_cards(header, **header_cards)
 
     return BinTableHDU(rad_max_table, header=header, name=extname)
+
+
+def compare_irf_cuts(files, extname):
+    """
+    Reads in a list of IRF files and checks if the same cuts have been applied in all of them
+
+    Parameters
+    ----------
+    files: list of strings
+        files to be read
+    extname: string
+        name of the extension with cut values to read the data from in fits file
+
+    Returns
+    -------
+    match: Boolean
+        if the cuts are the same in all the files
+    """
+    with fits.open(files[0]) as hdul0:
+        data0 = hdul0['THETA_CUTS'].data
+
+    for file_name in files[1:]:
+        with fits.open(file_name) as hdul:
+            data = hdul['THETA_CUTS'].data
```

Comment:
good point, thx, extname was a parameter but was not actually used, now I changed it so arbitrary extension name can be checked. 
About disentangling IO in this case I have mixed feelings, while it is very important to check that the files that are being read in are consistent, it is somehow hard to imagine that the same code generates a bunch of IRFs and then still needs to check that the cuts are consistent. Nevertheless, I will separate the two, also to simplify the tests as you wrote in the above comment.  I'm closing this one now, because IO is mentioned also in the comment above